shell/browser/api/electron_api_web_contents.cc

```diff
@@ -816,6 +816,9 @@ WebContents::WebContents(v8::Isolate* isolate,
   // Get type
   options.Get("type", &type_);
 
+  // Get transparent for guest view
+  options.Get("transparent", &transparent_);
```

Comment:
Let's rename this internal value as to not create confusion with passing down WebPreferences from a new BrowserWindow.
```suggestion
  options.Get("guestTransparent", &guest_transparent_);
```

Reply:
Added "guest" prefix